include/xtensa-dynconfig.h

```diff
@@ -15,75 +15,80 @@
    along with this program; if not, write to the Free Software
    Foundation, 51 Franklin Street - Fifth Floor, Boston, MA 02110-1301, USA.  */
 
-#ifndef XTENSA_DYNAMIC_CONFIG_H
-#define XTENSA_DYNAMIC_CONFIG_H
+#ifndef XTENSA_DYNCONFIG_H
+#define XTENSA_DYNCONFIG_H
+
+#include <stdarg.h>
+#include <stdio.h>
+#include "xtensa-config.h"
```

Comment:
Why are these needed? There's nothing in this header that depends on any of these headers.

Reply:
Removed, thanks!